components/accept-financial-contributions/StartAcceptingFinancialContributionsPage.tsx

```diff
@@ -0,0 +1,445 @@
+              />
+            </P>
+            {props.host.hostFeePercent !== null && (
+              <P mt={2}>
+                <FormattedMessage
+                  defaultMessage="<b>{ hostFeePercent }%</b> Host fee"
+                  values={{
+                    hostFeePercent: props.host.hostFeePercent,
+                    b: chunks => <strong>{chunks}</strong>,
+                  }}
+                />
+              </P>
+            )}
+          </Box>
+          {props.host.description !== null && props.host.description.length !== 0 && (
+            <React.Fragment>
+              <Flex mx={3} pt={3} alignItems="center">
+                <Span
+                  color="black.700"
+                  fontSize="12px"
+                  lineHeight="16px"
+                  textTransform="uppercase"
+                  fontWeight="500"
+                  letterSpacing="0.06em"
+                >
+                  <FormattedMessage id="OurPurpose" defaultMessage="Our purpose" />
+                </Span>
+                <StyledHr borderColor="black.300" flex="1 1" ml={2} />
+              </Flex>
+              <P mx={3} mt={1} fontSize="12px" lineHeight="18px" color="black.800">
+                {props.host.description}
+              </P>
+            </React.Fragment>
+          )}
+        </Box>
+        <Box mx={3} mt={3}>
+          <StyledButton
+            onClick={() => {
+              props.onHostApplyClick(props.host);
+              setShowApplyToHostModal(true);
+            }}
+            buttonStyle="primary"
+            width="100%"
+          >
+            <FormattedMessage id="Apply" defaultMessage="Apply" />
+          </StyledButton>
+        </Box>
+      </StyledCollectiveCardWrapper>
+      {showApplyToHostModal && (
+        <ApplyToHostModal
+          hostSlug={props.host.slug}
+          collective={props.collective}
+          onClose={() => setShowApplyToHostModal(false)}
+          onSuccess={() => {
+            return router
+              .push(`${props.collective.slug}/accept-financial-contributions/host/success`)
+              .then(() => window.scrollTo(0, 0))
+              .then(() => {
+                confettiFireworks(5000, { zIndex: 3000 });
+              });
+          }}
+        />
+      )}
+    </React.Fragment>
+  );
+}
+
+const FindAFiscalHostQuery = gql`
+  query FindAFiscalHostQuery($tags: [String], $limit: Int, $country: [CountryISO], $currency: String) {
+    hosts(tag: $tags, limit: $limit, tagSearchOperator: OR, country: $country, currency: $currency) {
+      totalCount
+      nodes {
+        id
+        legacyId
+        createdAt
+        settings
+        type
+        name
+        slug
+        description
+        longDescription
+        currency
+        totalHostedCollectives
+        hostFeePercent
+        isTrustedHost
+        location {
+          id
+          country
+        }
+        tags
+      }
+    }
+  }
+`;
+
+function FeaturedFiscalHostResults({
```

Comment:
This one too, `OtherFiscalHostResults` and `FindAHostSearch`. I think the file would be simpler to read if we try to do 1 component per file (except for simple `styled`).